src/Microphysics1M.jl

```diff
@@ -113,11 +113,15 @@ function radar_reflectivity(
     ρ::FT,
 ) where {FT}
 
-    n0 = get_n0(pdf)
-    λ = lambda(pdf, mass, q, ρ)
-    Z₀ = FT(1e-18)
+    # change units for accuracy
+    n0 = get_n0(pdf) * FT(1e-12)
+    λ = lambda(pdf, mass, q, ρ) * FT(1e-3)
 
-    return 10 * log10((720 * n0 / λ^7) / Z₀)
+    Z = (λ == FT(0)) ? FT(0) : (720 * n0 / λ^7)
```

Comment:
`== FT(0)` is not safe for float numbers. You should do `lambda < eps(FT)` instead